src/lib/onboarding/router.ts

```diff
@@ -97,6 +103,31 @@ router.post('/submit-verified', async (req, res) => {
   }
 });
 
+/// FALLBACK ONLY
+router.post('/submit-with-sig', async (req, res) => {
+  try {
+    const accountWithSig = req.body;
+
+    if (!validateAccountWithSig(accountWithSig))
```

Comment:
`validateAccountWithSig` sounds like it would throw if not valid
Whereas `isAccountWithSigValid` would perform a check.

Just a semantic nitpick

Reply:
agree fixed 236177fcd1438f5e30778db7ac52875d0d879aa8